website/pages/docs/connect/terminating_gateway.mdx

```diff
@@ -0,0 +1,132 @@
+hold certificates to decrypt Consul Connect traffic directed at them and may be configured with credentials to connect
+to linked services. Connections over the WAN or open internet should flow through [mesh gateways](/docs/connect/mesh_gateway)
+whenever possible since they are not capable of decrypting traffic or connecting directly to services.
+
+By specifying a path to a [CA file](/docs/agent/config-entries/terminating-gateway#cafile) connections
+from the terminating gateway will be encrypted using one-way TLS authentication. If a path to a
+[client certificate](/docs/agent/config-entries/terminating-gateway#certfile)
+and [private key](/docs/agent/config-entries/terminating-gateway#keyfile) are also specified connections
+from the terminating gateway will be encrypted using mutual TLS authentication.
+
+If none of these are provided, Consul will **only** encrypt connections to the gateway and not
+from the gateway to the destination service.
+
+-> **Note:** If certificates and keys are configured the terminating gateway will upgrade HTTP connections to TLS. 
+Client applications can issue plain HTTP requests even when connecting to servers that require HTTPS.
+
+## Prerequisites
+
+Each terminating gateway needs:
+
+1. A local Consul client agent to manage its configuration.
+2. General network connectivity to services within its local Consul datacenter.
+
+Terminating gateways also require that your Consul datacenters are configured correctly:
+
+- You'll need to use Consul version 1.8.0.
+- Consul [Connect](/docs/agent/options#connect) must be enabled on the datacenter's Consul servers.
+- [gRPC](/docs/agent/options#grpc_port) must be enabled on all client agents.
+
+Currently, [Envoy](https://www.envoyproxy.io/) is the only proxy with terminating gateway capabilities in Consul.
+
+- Terminating gateway proxies receive their configuration through Consul, which
+  automatically generates it based on the gateway's registration. Currently Consul
+  can only translate terminating gateway registration information into Envoy
+  configuration, therefore the proxies acting as terminating gateways must be Envoy.
+
+Connect proxies that send upstream traffic through a gateway aren't
+affected when you deploy terminating gateways. If you are using non-Envoy proxies as
+Connect proxies they will continue to work for traffic directed at services linked to
+a terminating gateway as long as they discover upstreams with the
+[/health/connect](/api/health#list-nodes-for-connect-capable-service) endpoint.
+
+## Running and Using a Terminating Gateway
+
+You must complete the following steps to configure a terminating gateway to proxy traffic from services in the Consul service mesh:
+
+1. On a host dedicated to the gateway and Consul client, start an Envoy proxy using the [envoy subcommand](/docs/commands/connect/envoy#terminating-gateways) and
```

Comment:
```suggestion
1. On a Consul client, that is also a dedicated host for the gateway, start an Envoy proxy using the [envoy subcommand](/docs/commands/connect/envoy#terminating-gateways) and
```

Reply:
I think this doesn't really work, since a Consul client is an agent and not a host.

Rewording as "On a host with a Consul client agent" to also be less opinionated about it needed to be dedicated to the gateway.